app/views/layouts/application.html.erb

```diff
@@ -19,7 +19,16 @@
        <%= "Logged in as #{current_user.name}" %> <br>
        <%= link_to "Profile Page", "/profile" %>
        <%= link_to  "logout", "/logout" %>
-       <%= link_to "Dashboard", 'merchant/dashboard' %>
+       <% if current_uri != @merchant_dash_uri %>
+        <%= link_to "Dashboard", 'merchant/dashboard' %>
+       <% end %>
+      <% elsif current_admin? %>
+      <%= "Logged in as #{current_user.name}" %> <br>
+      <%= link_to "Profile Page", "/profile" %>
+      <%= link_to  "logout", "/logout" %>
+      <% if current_uri != @admin_dash_uri %>
+       <%= link_to "Dashboard", 'admin/dashboard' %>
+      <% end %>
```

Comment:
Created these if statements to make sure the dashboard link was not available while on the dashboard.

Reply:
Nice Job